Dockerfile

```diff
@@ -1,15 +1,13 @@
-FROM registry.access.redhat.com/ubi8/nodejs-16
+FROM registry.access.redhat.com/ubi8/nodejs-18
```

Comment:
Since we are upgrading base image, is there any reason not to use `registry.access.redhat.com/ubi8/nodejs-18-minimal`? Like https://github.com/app-sre/qontract-server/blob/dc3452748211c36547182aeee54ccb15fe346a49/Dockerfile#L19

Reply:
@hemslo, nice idea.

I haven't tried the minimal before, so let me see how it fares. Let me try.

Reply:
Works fine and has the added benefits of being a lot smaller per:

```
<none>                                              <none>                    dc4c02c9709a   28 seconds ago   270MB
<none>                                              <none>                    7a2a15c0d7b3   6 seconds ago    764MB
```

Reply:
Done. Thank you!